Dockerfile

```diff
@@ -1,13 +1,17 @@
 FROM fedora:latest
 
 RUN dnf update -y
-RUN dnf install git xorg-x11-server-Xvfb procps-ng wget libjpeg-turbo-devel 'dnf-command(builddep)' -y
+RUN dnf install git xorg-x11-server-Xvfb procps-ng wget libjpeg-turbo-devel expat-devel 'dnf-command(builddep)' -y
 RUN dnf builddep gtk4 -y
 
 # build gtk4 from the latest release
-ADD https://download.gnome.org/sources/gtk/4.6/gtk-4.6.0.tar.xz /tmp/gtk-4.6.0.tar.xz
-RUN tar -xf /tmp/gtk-4.6.0.tar.xz --directory /tmp
-WORKDIR /tmp/gtk-4.6.0
+#ADD https://download.gnome.org/sources/gtk/4.6/gtk-4.6.0.tar.xz /tmp/gtk-4.6.0.tar.xz
+#RUN tar -xf /tmp/gtk-4.6.0.tar.xz --directory /tmp
+# WORKDIR /tmp/gtk-4.6.0
```

Comment:
Was it intended to merge these comments?

Reply:
yes

Reply:
I plan to switch again to 4.8.0 once it is released

Reply:
Okay :)